driver/makedepend.ml

```diff
@@ -225,15 +267,28 @@ let print_dependencies target_files deps =
   List.iter print_dep deps;
   print_string "\n"
 
+let is_predef dep =
+  (String.length dep > 0) &&
+  (
+    match dep.[0] with
+    | 'A'..'Z' | '\128'..'\255' -> true
+    | _ -> false
+  )
```

Comment:
I believe that the name is the wrong way around: this is `not_predef`, not `is_predef`. The "predef" name we want to avoid is `"*predef*"`, so it returns `false` for this function.